library/init/meta/interactive.lean

```diff
@@ -6,8 +6,11 @@ Authors: Leonardo de Moura
 prelude
 import init.meta.tactic init.meta.rewrite_tactic init.meta.simp_tactic
 import init.meta.smt.congruence_closure init.category.combinators
+import init.meta.lean_parser
 
 namespace interactive
+@[reducible] meta def parse {α : Type} (p : lean_parser α) : Type := α
```

Comment:
> λ e, `(%%e)

I believe this is the identity function.

> We need a pexpr at tactic parse time that will evaluate to a pexpr at tactic run time. I think this instance should just construct quote_macro(e)

You're completely right!  I missed the fact that we only parse pre-expressions to produce *quoted* pre-expressions.  We never insert a pre-expression back into the AST as-is.  

If we ever want to parse an argument both as-is and quoted (for whatever reason), we can still do the `antiquote_pexpr` typedef.